repos/rust-lang/surveys.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "surveys"
+description = "Repo for coordinating the creation, distribution, collection, and analysis of surveys for the Rust project. "
+bots = []
+
+[access.teams]
+community-survey = "maintain"
```

Comment:
We have a few repos with teams with maintain access. I think we might want to consider restricting this to admin. I don't believe there's anything that admins can't do that we want to allow. It's things like deleting the repo, transferring ownership, etc. Ideally the only team with maintain privileges is infra-admins.